gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/GemFireCacheImpl.java

```diff
@@ -283,9 +283,9 @@
   /**
    * System property to disable default snapshot
    */
-  public final boolean DEFAULT_SNAPSHOT_DISABLED = Boolean.getBoolean("gemfire.Cache.DISABLE_DEFAULT_SNAPSHOT_ISOLATION");
+  public final boolean DEFAULT_SNAPSHOT_ENABLED = Boolean.getBoolean("gemfire.Cache.ENABLE_DEFAULT_SNAPSHOT_ISOLATION");
 
-  public final boolean DEFAULT_SNAPSHOT_ENABLED_TX = Boolean.getBoolean("gemfire.Cache.DISABLE_DEFAULT_SNAPSHOT_ISOLATION_TX");
+  public final boolean DEFAULT_SNAPSHOT_ENABLED_TX = Boolean.getBoolean("gemfire.Cache.ENABLE_DEFAULT_SNAPSHOT_ISOLATION_TX");
```

Comment:
Use "gemfire.cache." for consistency with naming convention.

Also use SystemProperties.getServerInstance().getBoolean("cache.ENABLE_...") which allows changing the prefix as per the product being used (its "gemfire." for GemFire, "gemfirexd." for GemFireXD and we can change similarly for snappydata to be "snappydata.store." later).